packages/data-stores/src/onboard/actions.ts

```diff
@@ -314,6 +320,25 @@ export const setEditEmail = ( email: string ) => ( {
 	email,
 } );
 
+export const setDomainForm = ( step: Record< string, string > ) => {
+	const lastUpdated = Date.now();
+
+	return {
+		type: 'SET_DOMAIN_FORM' as const,
+		step: { ...step, lastUpdated },
+	};
+};
```

Comment:
We should create an issue to nuke every single state that is an untyped object. We should be able to track every piece of info.

Reply:
On it 👍